src/app/checklist/app-updates/app-updates.service.ts

```diff
@@ -2,7 +2,7 @@ import { ApplicationRef, Injectable } from '@angular/core';
 import { MatSnackBar } from '@angular/material';
 import { SwUpdate } from '@angular/service-worker';
 import { fromEvent as observableFromEvent, merge as observableMerge, Observable, of as observableOf } from 'rxjs';
```

Comment:
Why do you use aliases here for `fromEvent`, `merge`, `of` etc.?

Reply:
It was just more familiar to me when I was migrating to rxjs 6, It's not necessary now, I'll change it.